File: C#/Program22.cs
/*

Write a program which accept number from user and count frequency of such a 
digits which are less than 6.

Input : 2395 
Output : 3
Input : 1018
Output : 3

*/


using System;

class Demo{
 
    public int CountFrequencyOfLessThan6(int iNo){
     
        int iDigit  = 0;
        int iCnt = 0;

        while(iNo != 0){

            iDigit = iNo % 10;

              if(iDigit  < 6)
              {
                 iCnt++;
              }

            iNo = iNo / 10;

        }

       return iCnt;

    }

}

class program22{
       
     public static void Main(string[] args){

            int iNo = Convert.ToInt32(Console.ReadLine());
            
            Demo dobj = new Demo();

            int iRet = dobj.CountFrequencyOfLessThan6(iNo);

            
            Console.WriteLine(iRet);
            

     }
     
}

File: JAVA/Program22.java
/*

Accept number from user and display below pattern.
Input : 8
Output : 2 4 6 8 10 12 14 16

*/


import java.util.*;

class Demo22
{
	 public void Display(int iNo)
	 {
	 	int i = 0;

         System.out.print("\n");

	 	for(i = 1; i <= iNo; i++)
	 	{
	 		System.out.print(i*2+"\t");
	 		
	 	}
	 }

}

class Program22
{
	public static void main(String[] args) 
	{
	   Scanner sobj = new Scanner(System.in);
	   Demo22 dobj = new Demo22();

	   int iVal = 0;
       
       System.out.print("\nEnter Number : ");
       iVal = sobj.nextInt();

	   dobj.Display(iVal);

	   System.out.print("\n");

	}
}